ValveResourceFormat/IO/FileExtract.cs

```diff
@@ -2,14 +2,14 @@
 using System.IO;
 using System.Text;
 using SkiaSharp;
-using ValveResourceFormat;
+using SteamDatabase.ValvePak;
 using ValveResourceFormat.ResourceTypes;
 
 namespace ValveResourceFormat.IO
 {
     public static class FileExtract
     {
-        public static Span<byte> Extract(Resource resource)
+        public static Span<byte> Extract(Resource resource, Package package = null, string targetPath = null)
```

Comment:
I think a better design would be to change this to ienumerable and yield more resources if needed. Also why do you need to pass a package, it's already set on the exporter object, is it not?